demosplan/DemosPlanCoreBundle/Logic/Statement/AssessmentTableExporter/AssessmentTablePdfExporter.php

```diff
@@ -113,7 +113,17 @@ public function __invoke(array $parameters): array
             if (!array_key_exists('filters', $parameters) || !is_array($parameters['filters'])) {
                 $parameters['filters'] = [];
             }
-            $parameters = $this->addStatementsFromCurrentQueryHashToFilter($parameters, $procedureId, $original);
+            if (!array_key_exists('statementId', $parameters)) {
+                $parameters = $this->addStatementsFromCurrentQueryHashToFilter($parameters, $procedureId, $original);
+            } else {
+                /*
+                 * in case the key 'statementId' was set by the invoking
+                 * { @link AssessmentTableZipExporter::getAttachmentsOfStatements }
+                 * do not try to obtain the ids from session
+                 */
+                $parameters['items'] = [$parameters['statementId']];
```

Comment:
Param `sort` is not set here. Is this intended?

Reply:
done in [c576805](https://github.com/demos-europe/demosplan-core/pull/2444/commits/c5768052f92bda399a1945ead9598157b4cc75e2)